docker-compose.yml

```diff
@@ -89,6 +86,20 @@ services:
       POSTGRES_PASSWORD: "admin"
     volumes:
       - hapi-fhir-postgres:/var/lib/postgresql/data
+  keycloak-postgres:
+    image: postgres:15
+    container_name: keycloak-postgres
+    restart: always
+    ports:
+      - "5432:5432"
```

Comment:
Aren't both postgres containers using this port? Did you test this on your local system?

Reply:
I was initially testing with MySQL and PostgreSQL while running docker-compose up. During this process, I encountered some Keycloak errors, so I added the port mappings to test certain Keycloak changes separately. Unfortunately, those modifications were inadvertently committed.